google/cloud/aiplatform/datasets/tabular_dataset.py

```diff
@@ -15,16 +15,23 @@
 # limitations under the License.
 #
 
+import csv
 from typing import Optional, Sequence, Tuple, Union
 
 from google.auth import credentials as auth_credentials
 
+from google.cloud import bigquery
+from google.cloud import storage
+
 from google.cloud.aiplatform import datasets
 from google.cloud.aiplatform.datasets import _datasources
 from google.cloud.aiplatform import initializer
 from google.cloud.aiplatform import schema
 from google.cloud.aiplatform import utils
 
+from typing import List
```

Comment:
This should be imported above with the rest of the types.

Reply:
Ah no lint check?